packages/drng/collective_beacon_payload.go

```diff
@@ -109,7 +106,7 @@ func FromBytes(bytes []byte, optionalTargetObject ...*Payload) (result *Payload,
 }
 
 // Bytes returns the collective beacon payload bytes.
-func (payload *Payload) Bytes() (bytes []byte) {
+func (payload *CollectiveBeaconPayload) Bytes() (bytes []byte) {
```

Comment:
Rename the method receiver to a single character. Like this:
```suggestion
func (p *CollectiveBeaconPayload) Bytes() (bytes []byte) {
```